### PR TITLE
add iframe

### PR DESCRIPTION
**Pull request by ori-shalom**

### CODE REVIEW

File: sdk/forms/test/forms.spec.ts [hunk 1]
```diff
@@ -1,82 +1,82 @@
...
-    // expect all values to be tokens
-    expect(value).toMatch(/^[0-9a-z]{8}-([0-9a-z]{4}-){3}[0-9a-z]{12}$/)
-  }
-
-  const consoleMessage = (await consoleEvent).text();
-  // The original submit handler (which log the form data to the logs) should have been called and should receive the form data tokenized.
-  expect(consoleMessage).toMatch(/my original on submit handler/g)
-  expect(consoleMessage).toMatch(/card_number: [0-9a-z]{8}-([0-9a-z]{4}-){3}[0-9a-z]{12}/g)
-});
+// import {expect, test} from '@playwright/test';
```
Comment (motymichaely) [inline sdk/forms/test/forms.spec.ts:1]: @ori-shalom Delete this file?

File: sdk/forms/vite.config.ts [hunk 2]
```diff
@@ -1,111 +1,80 @@
...
+import { initDevelopmentVault } from './vite/init-dev-vault';
+import { version } from './package.json';
 
 // https://vitejs.dev/config/
-export default defineConfig((async () => {
-  let devVaultPort = 8123;
-  if (process.env.NODE_ENV === 'development') {
-    devVaultPort = await initDevelopmentVault();
+const config: UserConfigFnPromise = async ({ mode }): Promise<UserConfig> => {
+  if (mode === 'dev') {
```
Comment (motymichaely) [inline sdk/forms/vite.config.ts:11]: @ori-shalom Would this be publicly available?

**Review by motymichaely — COMMENTED** [review]
@ori-shalom LGTM, added a few comments and questions. Thanks!

File: sdk/forms/src/protected-forms/iframe/validations.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
+const regexp = {
```
[the remark this answers is not in the document]
Comment (ori-shalom) [reply sdk/forms/src/protected-forms/iframe/validations.ts:1]: It's usually better for UX to perform validation on the frontend side.
For example once the field become valid again we can show indication it is valid and don't need to send request to the server for that.
I wrote these validations based on the Go code so they will be aligned.

File: sdk/forms/vite.config.ts (hunk 2 above)
Comment (ori-shalom) [reply sdk/forms/vite.config.ts:11]: WDYM? This is only vite config used for bundling and local dev server. It's not delivered with the final artifacts.

**Review by david-piiano — APPROVED** [review]

**Pull request merged**